internal/controller/controllers/bmh_agent_controller.go

```diff
@@ -755,6 +775,16 @@ func (r *BMACReconciler) reconcileSpokeBMH(ctx context.Context, log logrus.Field
 		Namespace: agent.Spec.ClusterDeploymentName.Namespace,
 		Name:      fmt.Sprintf(adminKubeConfigStringTemplate, cd.Name),
 	}
+	isNonePlatform, err := r.isNonePlatform(ctx, cd)
+	if err != nil {
+		log.WithError(err).Errorf("Failed to determine if platform is none for cluster deployment %s/%s", cd.Namespace, cd.Name)
+	}
+	if isNonePlatform || err != nil {
+		// If none platform do not attempt to create spoke BMH
+		// If error occurred no retry will be attempted since we will get same result
```

Comment:
There will be a retry if the `ClusterDeployment` changes since BMAC has a watch on this CRD.

Could you mention this? (just asking this because we need another push to remove `DEFAUAT_BMH_REQUEUE_DURATION` )